cfme/common/vm.py

```diff
@@ -577,8 +577,7 @@ def create(self, vm_name, provider, form_values=None, cancel=False, check_existi
                 if provision_request.is_succeeded(method='ui'):
                     logger.info('Waiting for vm %s to appear on provider %s', vm.name,
                                 provider.key)
-                    wait_for(provider.mgmt.does_vm_exist, [vm.name],
-                             handle_exception=True, num_sec=600)
+                    wait_for(provider.mgmt.does_vm_exist, [vm.name], num_sec=600)
```

Comment:
Question: Why are you removing `handle_exception`?

Reply:
I don't think that's a good change to make in the common method for create. Some does_vm_exist methods may raise on some providers, and some may just return None/False. If, in a specific use for a test, you want this to raise exception on the very first call to this method, then you should do it custom in the test. You can pass `wait=False` and `find_in_cfme=False` when calling to have it return, and you can call this method yourself.

Reply:
Well if `does_vm_exist` can raise on some and return False on other , then I would call this a problem to solve.